debezium-core/src/main/java/io/debezium/transforms/PrepareCloudEvent.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms;
+
+import static org.apache.kafka.connect.transforms.util.Requirements.requireStruct;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.json.JsonConverter;
+import org.apache.kafka.connect.json.JsonConverterConfig;
+import org.apache.kafka.connect.transforms.Transformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import io.debezium.converters.spi.CloudEventsMaker;
+import io.debezium.transforms.outbox.AdditionalFieldsValidator;
+
+/**
+ * The transform converts a CloudEvent to a structure suitable for `JdbcSinkConnector`. It uses provided by a user
+ * mapping between a CloudEvent's fields and names of database columns. The resulting value schema has no name. A
+ * CloudEvent's `data` field is flattened if needed
+ *
+ * @author Roman Kudryashov
+ */
+public class PrepareCloudEvent<R extends ConnectRecord<R>> implements Transformation<R> {
```

Comment:
Hi, the SMT name is a bit too generic and it realy does not describe its purpose.
Also is it a generic SMT or it is useful for JDBC Sink connector only? If the latter then I propose to move it to that repository.